app/views/upcoming_offline_reservation_mailer/generate_mail.html.haml

```diff
@@ -0,0 +1,10 @@
+%p
```

Comment:
`generate_mail` is a little ambiguous. Is there a more precise name we can come up with?